### PR TITLE
Added toasting to Questions, Queue, and Queues Components

### PR DESCRIPTION
**Pull request by KeenWill**

### CODE REVIEW

File: src/app/pages/queue/question/question.component.ts [hunk 1]
```diff
@@ -28,19 +33,19 @@ export class QuestionComponent {
 
   deleteQuestion(id: string) {
     // this.questionService.deleteQuestion(id);
-    console.log('deleting question');
+    this.showToast('Deleted Question', !!this.question.title ? this.question.title : undefined);
```
Comment (dlike230) [inline src/app/pages/queue/question/question.component.ts]:
```suggestion
    this.showToast('Deleted Question', this.question.title || null);
```

File: src/app/pages/queue/question/question.component.ts [hunk 2]
```diff
@@ -28,19 +33,19 @@ export class QuestionComponent {
...
   }
 
   answerQuestion(id: string) {
     // this.questionService.deleteQuestion(id);
     this.questionService.answerQuestion(id, this.question.queueId, this.user.uid);
-    console.log('answering question');
+    this.showToast('Answered Question', !!this.question.title ? this.question.title : undefined);
```
Comment (dlike230) [inline src/app/pages/queue/question/question.component.ts]:
```suggestion
    this.showToast('Answered Question', this.question.title || null);
```

File: src/app/pages/queue/question/question.component.ts [hunk 3]
```diff
@@ -28,19 +33,19 @@ export class QuestionComponent {
...
   }
 
   unanswerQuestion(id: string) {
     // this.questionService.deleteQuestion(id);
     this.questionService.unanswerQuestion(id, this.question.queueId);
-    console.log('unanswering question');
+    this.showToast('Unanswered Question', !!this.question.title ? this.question.title : undefined);
```
Comment (dlike230) [inline src/app/pages/queue/question/question.component.ts]:
```suggestion
    this.showToast('Unanswered Question', this.question.title || null);
```

File: src/app/pages/queue/question/question.component.ts [hunk 4]
```diff
@@ -50,10 +55,15 @@ export class QuestionComponent {
...
     dialogRef.afterClosed().subscribe(result => {
+      this.showToast('Updated Question');
       result.question.desc = result.desc;
       result.question.title = result.title;
       this.questionService.updateQuestion(result);
     });
   }
 
+  showToast(title: string, message?: string): void {
+    this.toastrService.show(!!message ? message : '', title, { limit: 3, status: 'success' });
```
Comment (dlike230) [inline src/app/pages/queue/question/question.component.ts]:
```suggestion
    this.toastrService.show(message || '', title, { limit: 3, status: 'success' });
```

File: src/app/pages/queue/queue/queue.component.ts [hunk 5]
```diff
@@ -38,41 +41,47 @@ export class QueueComponent implements OnInit {
...
   askQuestion(asker: User) {
+    this.showToast('Asked Question Answered Questions');
     const dialogRef = this.dialog.open(AskQuestionDialogComponent, {
       width: '250px',
       data: {},
     });
 
     dialogRef.afterClosed().subscribe(result => {
       this.questionService.createQuestion(asker, this.queue.id, result.title || '', result.desc || '');
+      this.showToast('Asked Question', !!result.title ? result.title : undefined);
```
Comment (dlike230) [inline src/app/pages/queue/queue/queue.component.ts]:
```suggestion
      this.showToast('Asked Question', result.title || null);
```

File: src/app/pages/queue/queue/queue.component.ts [hunk 6]
```diff
@@ -38,41 +41,47 @@ export class QueueComponent implements OnInit {
...
     this.queuesService.closeQueue(this.queue.id);
   }
 
   deleteQueue() {
+    this.showToast('Deleted Queue');
     this.queuesService.deleteQueue(this.queue.id);
   }
 
+  showToast(title: string, message?: string): void {
+    this.toastrService.show(!!message ? message : '', title, { limit: 3, status: 'success' });
```
Comment (dlike230) [inline src/app/pages/queue/queue/queue.component.ts]:
```suggestion
    this.toastrService.show(message || '', title, { limit: 3, status: 'success' });
```

File: src/app/pages/queue/queues/queues.component.ts [hunk 7]
```diff
@@ -44,8 +50,13 @@ export class QueuesComponent implements OnInit {
 
     dialogRef.afterClosed().subscribe(result => {
       this.queuesService.createQueue(result.name || '', result.desc || '', result.allowGrouping || false);
+      this.showToast('Created New Queue', !!result.name ? result.name : undefined);
```
Comment (dlike230) [inline src/app/pages/queue/queues/queues.component.ts]:
```suggestion
      this.showToast('Created New Queue', result.name || null);
```

File: src/app/pages/queue/question/question.component.ts (hunk 1 above)
Comment (KeenWill) [reply src/app/pages/queue/question/question.component.ts]: I was considering this...

I was thinking that i would need to specify undefined to make sure it doesn't fill in the optional parameter with null.

File: src/app/pages/queue/question/question.component.ts (hunk 2 above)
Comment (KeenWill) [reply src/app/pages/queue/question/question.component.ts]: See https://github.com/KeenWill/OHQueue/pull/28#discussion_r340955285

File: src/app/pages/queue/question/question.component.ts (hunk 3 above)
Comment (KeenWill) [reply src/app/pages/queue/question/question.component.ts]: See https://github.com/KeenWill/OHQueue/pull/28#discussion_r340955285

File: src/app/pages/queue/question/question.component.ts (hunk 4 above)
Comment (KeenWill) [reply src/app/pages/queue/question/question.component.ts]: See https://github.com/KeenWill/OHQueue/pull/28#discussion_r340955285

File: src/app/pages/queue/queue/queue.component.ts (hunk 5 above)
Comment (KeenWill) [reply src/app/pages/queue/queue/queue.component.ts]: See https://github.com/KeenWill/OHQueue/pull/28#discussion_r340955285

File: src/app/pages/queue/queue/queue.component.ts (hunk 6 above)
Comment (KeenWill) [reply src/app/pages/queue/queue/queue.component.ts]: See https://github.com/KeenWill/OHQueue/pull/28#discussion_r340955285

File: src/app/pages/queue/queues/queues.component.ts (hunk 7 above)
Comment (KeenWill) [reply src/app/pages/queue/queues/queues.component.ts]: See https://github.com/KeenWill/OHQueue/pull/28#discussion_r340955285

**Pull request merged**